tic-tac-toe.js

```diff
@@ -1,7 +1,121 @@
+$(document).on('ready', function() {
+
+  var game = new TicTacToe();
+
+  function updateMessage(message) {
+    $('#message').text(message)
+  }
+
+  $('td').hover(
+       function(){ $(this).addClass('space_hover') },
+       function(){ $(this).removeClass('space_hover') }
+  )
+
+  $('td').click(function() {
+    var space = $(this)
+    var board = $('#game-board')
+
+    if (game.isSpaceEmpty(space)) {
+      if (game._current_player === "Player 2") {
+        space.addClass("player_2")
+        if (game.checkIfWon(board,"player_2")) {
+          alert(game._current_player + " won!")
+          game.reset(board,game)
+          updateMessage("Your turn, " + game._current_player)
+        } else {
+          if (game.checkIfDraw(board)) {
+            alert("It's a draw!")
+            game.reset(board,game)
+            updateMessage("Your turn, " + game._current_player)
+          }
+        game.changePlayer()
+        updateMessage("Your turn, " + game._current_player)
+        }
+      } else {
+        space.addClass("player_1")
+        if (game.checkIfWon(board,"player_1")) {
+          alert(game._current_player + " won!")
+          game.reset(board,game)
+          updateMessage("Your turn, " + game._current_player)
+        } else {
+          if (game.checkIfDraw(board)) {
+            alert("It's a draw!")
+            game.reset(board,game)
+            updateMessage("Your turn, " + game._current_player)
+          }
+        game.changePlayer()
+        updateMessage("Your turn, " + game._current_player)
+        }
```

Comment:
I noticed from line 18 to here there is some repetition depending on whose turn it is. I wonder if a helper method could have been created to reduce the repetition. Like a `play_game` method or something.